src/cli/domain/package-server-script/bundle/config/index.js

```diff
@@ -0,0 +1,63 @@
+/*jshint camelcase:false */
+'use strict';
+
+var webpack = require('webpack');
+var path = require('path');
+var wrapLoops = require('./wrapLoops');
+var externalDependenciesHandlers = require('./externalDependenciesHandlers');
+
+module.exports = function webpackConfigGenerator(params){
+  return {
+    entry: params.dataPath,
+    target: 'node',
+    output: {
+      path: '/build',
+      filename: params.fileName,
+      libraryTarget: 'commonjs2'
+    },
+    externals: externalDependenciesHandlers(params.dependencies),
+    module: {
+      loaders: [
+        {
+          test: /\.json$/,
+          exclude: /node_modules/,
+          loader: 'json-loader'
+        },
+        {
+          test: /\.js?$/,
+          exclude: /node_modules/,
+          loaders: [
+            'falafel-loader',
+            'babel-loader?' + JSON.stringify({
+              cacheDirectory: true,
+              'presets': [
+                [require.resolve('babel-preset-env'), {
+                  'targets': {
+                    'node': 0.10
```

Comment:
I know. This should be a short term pain though. I was planning to update this with the wepack2 upgrade as by tame we'll merge that mean we don't have any more 0.10&co to support. After that we my want to consider adding some global config to oc where you could set that target info there. Thoughts?